tests/nat/agent/test_base.py

```
@@ -41,9 +41,9 @@ def __init__(self, detailed_logs=True, log_response_max_chars=1000):
         self.detailed_logs = detailed_logs
         self.log_response_max_chars = log_response_max_chars
 
-    async def _build_graph(self, state_schema: type) -> CompiledGraph:
+    async def _build_graph(self, state_schema: type) -> CompiledStateGraph:
```

Ruff (0.12.2)

Unused method argument: `state_schema`

(ARG002)